NitroxClient/GameLogic/Terrain.cs

```diff
@@ -34,6 +34,7 @@ public void CellLoaded(Int3 batchId, Int3 cellId, int level)
 
         if (!visibleCells.Contains(cell))
         {
+            removedCells.Remove(cell);
```

Comment:
This seems more complete and how it should be but does this fix any bugs we have?